### PR TITLE
small team infopanel

### PR DESCRIPTION
**Pull request by zanderz**
Tested on mobile and desktop.
Cecile says the saved animation is in a separate ticket.
@keybase/react-hackers

### CODE REVIEW

File: shared/teams/really-leave-team/container-chat.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// @flow
```
Comment (zanderz) [inline shared/teams/really-leave-team/container-chat.js]: Marco suggested I copy this - it is the same as container.js except for onLeave navigation

File: shared/chat/conversation/info-panel/index.js [hunk 2]
```diff
@@ -82,13 +83,13 @@ type infoPanelProps = {
...
 
-type SmallTeamInfoPanelProps = infoPanelProps & {
+type ConversationInfoPanelProps = infoPanelProps & {
   onShowBlockConversationDialog: () => void,
   onShowNewTeamDialog: () => void,
   showTeamButton: boolean,
 }
 
-const _SmallTeamInfoPanel = (props: SmallTeamInfoPanelProps) => (
+const _ConversationInfoPanel = (props: ConversationInfoPanelProps) => (
```
Comment (zanderz) [inline shared/chat/conversation/info-panel/index.js]: Copied SmallTeamInfoPanel and renamed it ConversationInfoPanel

Comment (akalin-keybase) [reply shared/chat/conversation/info-panel/index.js]: Ah, got it, we're making a distinction between small teams, big teams, and old-style conversations.

**Comment by akalin-keybase** [comment]
There's a weird gap on desktop with small teams, though. That can be fixed in a follow up, though, I think.
screen shot 2017-11-20 at 3 08 35 pm

**Pull request merged**